packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/document/dto/UploadDocumentSuccessPayload.java

```diff
@@ -23,5 +24,5 @@
  *
  * @author sbegaudeau
  */
-public record UploadDocumentSuccessPayload(@NotNull UUID id, @NotNull DocumentDTO document, String report) implements IPayload {
+public record UploadDocumentSuccessPayload(@NotNull UUID id, @NotNull DocumentDTO document, String report, Map<String, String> idMapping) implements IPayload {
```

Comment:
This is returned to the frontend, it's quite odd to expose this information here. I'll see if we really need this part.

Reply:
After a review of the code it seems that it's the easiest way to achieve our goal. I'll add a small note here to indicate that we should probably consider removing this when we will improve the upload process which is a bit odd right now.